### ISSUE TITLE
Coordinate string format not working correctly with .NET Core 3.1 in rare cases.

### ISSUE THREAD

**Issue by Tronald** [Tronald/CoordinateSharp#147]
**Describe the bug**
Degrees Decimal Minute will sometimes improperly format a coordinate string if a value part is near 0.

**To Reproduce**
Run the following in .NET Core 3.1

```C#
Coordinate c = new Coordinate(65.25, -15.1859);
c.FormatOptions.Format = CoordinateFormatType.Degree_Minutes_Seconds;
Console.WriteLine(c.ToString());
```

**Expected behavior**
.NET FRAMEWORK: N 65º 15' 0" W 15º 11' 9.24" (CORRECT)
.NET Core: N 65º 15' -0" W 15º 11' 9.24" (INCORRECT).

**Environment (please complete the following information):**
 - .NET Core 3.1
 - CoordinateSharp 2.6.2.1

**Additional context**
1. Investigate `double.ToString()` behavior differences between Core and Framework.
2. Implement work around.
3. Report issue on .NET Core if needed.

**Comment by Tronald** [comment]
This appears to be specific to doubles in .NET Core 3.x. 

```c#
double d = -.1;
Console.WriteLine(Math.Round(d,0)); //-0 in .NET Core 3.x.
```

**Comment by Tronald** [comment]
_Roundtripping_ checks will be put in where necessary to convert -0 to 0 as needed. This should be a minimal performance hit, and appears to be the only workaround to the issue as MS did not build in an override to the behavior.

Will be fixed with next release.

**Comment by Tronald** [comment]
Fixed with [v2.7.1.1](https://github.com/Tronald/CoordinateSharp/releases/tag/v2.7.1.1) release on 12-JUL-2020.
